Task_functions/index.js

```diff
@@ -1,11 +1,13 @@
-//Write code for the 1st task below this
-
-
-//Write your code above this line
-
-
-
-//Write code for the 2nd task below this
-
```

Comment:
The parameter order in the 'greet' function seems to be incorrect. It's best practice to have the default parameter at the end.